WcaOnRails/app/jobs/submit_results_nag_job.rb

```diff
@@ -3,7 +3,7 @@ class SubmitResultsNagJob < ActiveJob::Base
   queue_as :default
 
   def nag_needed(competition)
-    (competition.results_nag_sent_at || competition.end_date) <= 1.week.ago
```

Comment:
@Luis-J-Ianez who had this idea at first. Does this look good? Or was it meant to be an early alert?